apps/skde/package.json

```diff
@@ -20,6 +20,9 @@
     "check-types": "tsc --noEmit",
     "start-server": "npx serve out/"
   },
+  "resolutions": {
+    "styled-components": "^5"
```

Comment:
Må vi bruke versjon 5? Har vi prøvd versjon 6? Burde kanskje være eksplisitt på versjonsnummer (`6.1.8`)?

Reply:
Mulig siste setning ikke er relevant for `resolutions`? Vet egentlig ikke hva det betyr.

Reply:
Har fjernet "resolutions". `styled-components` var allerede inkludert under "dependencies".